### PR TITLE
ref(icon refactor): replace icon print and update recovery code page

### PR DESCRIPTION
**Pull request by doralchan**
**Before:** 
Screen Shot 2020-03-30 at 4 47 42 PM

**After:**
Screen Shot 2020-03-30 at 4 49 51 PM

### CODE REVIEW

File: tests/js/spec/views/accountSecurityDetails.spec.jsx [hunk 1]
```diff
@@ -185,7 +185,7 @@ describe('AccountSecurityDetails', function() {
       expect(wrapper.find(downloadCodes)).toHaveLength(1);
       wrapper.find(downloadCodes).simulate('click');
 
-      expect(wrapper.find('Button InlineSvg[src="icon-print"]')).toHaveLength(1);
+      expect(wrapper.find('Button IconPrint')).toHaveLength(0);
```
Comment (billyvg) [inline tests/js/spec/views/accountSecurityDetails.spec.jsx]: Shouldn't this show up?

Comment (doralchan) [reply tests/js/spec/views/accountSecurityDetails.spec.jsx]: Yep - fixed it

File: src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/recoveryCodes.jsx [hunk 2]
```diff
@@ -47,19 +47,20 @@ class RecoveryCodes extends React.Component {
 
           <Actions>
             <Clipboard hideUnsupported value={formattedCodes}>
-              <Button size="small">
-                <InlineSvg src="icon-copy" />
+              <Button size="small" label="copy">
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/recoveryCodes.jsx]: We should translate these strings

```suggestion
              <Button size="small" label={t('copy'}>
```

File: tests/js/spec/views/accountSecurityDetails.spec.jsx [hunk 3]
```diff
@@ -185,7 +185,7 @@ describe('AccountSecurityDetails', function() {
       expect(wrapper.find(downloadCodes)).toHaveLength(1);
       wrapper.find(downloadCodes).simulate('click');
 
-      expect(wrapper.find('Button InlineSvg[src="icon-print"]')).toHaveLength(1);
+      expect(wrapper.find('Button[label="print"]')).toHaveLength(1);
```
Comment (billyvg) [inline tests/js/spec/views/accountSecurityDetails.spec.jsx]: Ooh this is a good improvement over the previous test , I would go a bit deeper down the tree and search for `button[aria-label=""]`

Comment (doralchan) [reply tests/js/spec/views/accountSecurityDetails.spec.jsx]: Ah good idea!

**Review by billyvg — APPROVED** [review]
thanks!

**Pull request merged**